features/src/main/scala/com/salesforce/op/features/FeatureSparkTypes.scala

```diff
@@ -263,12 +263,7 @@ case object FeatureSparkTypes {
    */
   def udf1[I <: FeatureType : TypeTag, O <: FeatureType : TypeTag](
     f: I => O
-  ): UserDefinedFunction = {
-    val inputTypes = Some(FeatureSparkTypes.sparkTypeOf[I] :: Nil)
-    val outputType = FeatureSparkTypes.sparkTypeOf[O]
-    val func = transform1[I, O](f)
-    UserDefinedFunction(func, outputType, inputTypes)
-  }
+  ): UserDefinedFunction = udf(transform1[I, O](f))
```

Comment:
I think there was a reason why udf-s were done this way, but I dont recall the reason. I guess it will either work, or you would discover why ;-)